python/ml4ir/applications/ranking/model/losses/listwise_losses.py

```diff
@@ -91,6 +91,99 @@ def final_activation_op(self, inputs, training=None):
         return self.final_activation_fn(logits)
 
 
+class AuxiliaryOneHotCrossEntropy(SoftmaxCrossEntropy):
+    """
+    Compute the one-hot softmax cross entropy loss on the auxiliary label
+    """
+
+    def call(self, inputs, y_true, y_pred, training=None):
```

Comment:
@mohazahran The function definitions are now different because they inherit from tensorflow layers itself.